infinispan/src/main/java/org/hibernate/ogm/datastore/infinispan/InfinispanDialect.java

```diff
@@ -55,9 +55,9 @@
  */
 public class InfinispanDialect<EK,AK,ISK> extends BaseGridDialect {
```

Comment:
Since this class relies on an embedded provider, maybe we'd better rename this to `InifinispanEmbeddedDialect`? If it's a matter of not breaking compatibility, we can always add another deprecated class with the legacy name (`InfinispanDialect`), extending this one.
Of course, if this class is ultimately expected to be used for both embedded and remote datastore providers, and we just don't have an interface for `InfinispanDatastoreProvider` yet, then it's okay.